.github/workflows/test-action-tuf.yaml

```diff
@@ -85,6 +85,33 @@ jobs:
       run: |
         cosign verify --rekor-url ${{ env.REKOR_URL }} --allow-insecure-registry ${{ env.demoimage }}
 
+    - name: Checkout TSA for testing.
+      uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+      with:
+        repository: sigstore/timestamp-authority
+        path: ./src/github.com/sigstore/timestamp-authority
+    - name: Build timestamp-cli
+      working-directory: ./src/github.com/sigstore/timestamp-authority
+      run: |
+        go build -o ./timestamp-cli ./cmd/timestamp-cli
+    - name: Exercise TSA
+      working-directory: ./src/github.com/sigstore/timestamp-authority
+      run: |
+        curl ${{ env.TSA_URL }}/api/v1/timestamp/certchain > ts_chain.pem
```

Comment:
nit: should this be in a shell script file, so it can be called by `test-action-tuf.yaml` and `test-release.yaml`, without duplicating it?

Reply:
That's a great idea!

Reply:
hm. The only problem is that we don't have the artifact in some of these tests (like the action) since we don't actually grab the repo, and the same for testing the release.
So we would have to either:
 * check out the repo to get the bash script
 * make the bash script to a release artifact that we can pull
 
 WDYT about doing this separately?

Reply:
separately sgtm